app/helpers/ems_physical_infra_helper/textual_summary.rb

```diff
@@ -14,7 +14,7 @@ def textual_group_properties
   def textual_group_relationships
     TextualGroup.new(
       _("Relationships"),
-      %i(physical_servers datastores vms hosts)
+      %i(physical_servers datastores vms physical_servers_with_host)
```

Comment:
I have renamed the relationship appropriately here, that's all.

So `hosts` would mean "Hosts", which is incorrect.
Since, it's really a relationship with `Physical Servers` that have `hosts`

Maybe this topology screenshot will clarify the nature of this relationship:
https://user-images.githubusercontent.com/1538216/26905292-6ba8d4ec-4b9b-11e7-91bd-77f80456cbbb.png